bin/terraform-deploy

```diff
@@ -36,12 +37,45 @@ else
   exit 1
 fi
 
-echo "-> Checking out terraform workspace $TERRAFORM_WORKSPACE …"
-terraform workspace select "$TERRAFORM_WORKSPACE"
-
-echo "-> Checking out and pulling branch $GIT_BRANCH …"
+echo "-> Checking out and pulling branch ${GIT_BRANCH}…"
 git checkout $GIT_BRANCH
 git pull origin $GIT_BRANCH
 
-echo "-> Applying Terraform command on the $TERRAFORM_WORKSPACE workspace …"
+echo "-> Checking for uncommitted changes…"
+
+GIT_DIFF=`git diff-index --name-status HEAD --`
+GIT_UNTRACKED=`git ls-files --exclude-standard --others --`
+
+if [ ! "$GIT_DIFF" = "" ]; then
+  cat <<EOF
+
+Changes found:
+
+$GIT_DIFF
+EOF
+fi
+
+if [ ! "$GIT_UNTRACKED" = "" ]; then
+  cat <<EOF
+
+Untracked files found:
+
+$GIT_UNTRACKED
+EOF
+fi
+
+if [ ! "$GIT_DIFF" = "" ] || [ ! "$GIT_UNTRACKED" = "" ]; then
+  echo
+  read -r -p "Continue? [y/N] " CONT
```

Comment:
Should we stick with the Terraform convention of only accepting `yes` to continue?  I know this is potentially less destructive than doing a Terraform deploy, but it at least follows the way we do things in the rest of the pipeline.

Reply:
Change made.